plugins/haconfig/keepalived/helpers.go

```diff
@@ -0,0 +1,78 @@
+package keepalived
+
+import (
+	"fmt"
+	"strings"
+
+	kclient "github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+type ReporterFunction func(string, error) 
+
+
+//  Delete matching replication controllers.
+func deleteReplicationControllers(name string, namespace string, selector map[string]string, kClient kclient.Interface, after ReporterFunction) {
+	labels := labels.SelectorFromSet(selector)
```

Comment:
So I had a conversation with @abhgupta yesterday and was asking re: the cleanup/deletion of these objects and how to go about doing it. The ```--delete``` option cleans up the deployment artifacts and uses this.